controllers/propagator/propagation.go

```diff
@@ -544,30 +576,45 @@ func (r *PolicyReconciler) handleRootPolicy(instance *policiesv1.Policy) error {
 	if err != nil {
 		reqLogger.Error(err, "Giving up on updating the root policy status...")
 		r.recordWarning(instance, "Failed to update the policy status")
+
 		return err
 	}
 
 	err = r.cleanUpOrphanedRplPolicies(instance, allDecisions)
 	if err != nil {
 		reqLogger.Error(err, "Giving up on deleting the orphaned replicated policies...")
 		r.recordWarning(instance, "Failed to delete orphaned replicated policies")
+
 		return err
 	}
 
 	reqLogger.Info("Reconciliation complete.")
+
 	return nil
 }
 
 // getApplicationPlacementDecisions return the placement decisions from an application
 // lifecycle placementrule
-func getApplicationPlacementDecisions(c client.Client, pb policiesv1.PlacementBinding, instance *policiesv1.Policy) ([]appsv1.PlacementDecision, *policiesv1.Placement, error) {
+func getApplicationPlacementDecisions(
+	c client.Client, pb policiesv1.PlacementBinding, instance *policiesv1.Policy,
+) ([]appsv1.PlacementDecision, *policiesv1.Placement, error) {
```

Comment:
I like this updated format for long function signatures. Is it automatic from a linter? Or is it done manually to resolve line length linting errors?

Reply:
This is manual unfortunately. This approach is very similar to how the black Python auto-formatting tool does things, so I just carried that over to my Go code.